pkg/secrets/secrets.go

```diff
@@ -38,11 +39,12 @@ func init() {
 // Init initializes the command and other options of the secrets package. Since
 // this package is used by the 'config' package to decrypt itself we can't
 // directly use it.
-func Init(command string, arguments []string, timeout int, maxSize int) {
+func Init(command string, arguments []string, timeout int, maxSize int, groupExecPerm bool) {
 	secretBackendCommand = command
 	secretBackendArguments = arguments
 	secretBackendTimeout = timeout
 	SecretBackendOutputMaxSize = maxSize
+	secretBackendCommandAllowGroupExec = groupExecPerm
```

Comment:
Could we log something when this is set to true ? Do we want to return on error when we don't run in a containerized env ?

Reply:
I think the log is not properly configure when we call the `Init()` function the `secret` package.
If we do at exec time, it will be quite verbose if they are decoding lot of secrets.

Reply:
it should be fine to `log` before the log package is initialized, when that happens the `log` package buffers the logs and actually flushes them once it's been initialized.

Logging a warning in non-containerized envs would make sense to me